source/release-notes-pxc-v8.0.27.rst

```diff
@@ -0,0 +1,50 @@
+.. _pdpxc.release_notes_8.0.27:
+
+********************************************************************************
+|pdmysql| 8.0.27 using |PXC|
+********************************************************************************
+
+:Date: , 2022
+:Installation: :ref:`installation`
+
+|pdmysql| is a single solution with the best and most critical enterprise components from the |mysql| open source community, designed and tested to work together. 
+
+|pdmysql| provides two deployment variants: one is based on |PS| and another one is based on |PXC|. 
+
+This release of |pdmysql| is focused on the |PXC|-based deployment variant. It is based on `Percona XtraDB Cluster 8.0.27-17.1 <https://www.percona.com/doc/percona-xtradb-cluster/8.0/release-notes/Percona-XtraDB-Cluster-8.0.27-17.1.html>`_ 
+
+Release Highlights
+==================
+
+The following lists a number of the bug fixes for *MySQL* 8.0.27, provided by Oracle, and included in Percona Server for MySQL:
```

Comment:
Would it make sense to include a link to the upstream MySQL 8.0.27 release notes here? This way, users could quickly check what other upstream changes have occurred, without having to check the PS 8.0.27 release notes first.

Reply:
Added